test/e2e-common.sh

```diff
@@ -55,6 +55,7 @@ function knative_setup() {
 
 # Setup resources common to all eventing tests.
 function test_setup() {
+  kubectl apply -f $(dirname $0)/e2e/config/istio-patch/istio-knative-extras.yaml
```

Comment:
maybe create a function istio_setup, and add this there?

Reply:
@chizhg do you know if we are doing something like this in eventing?

Reply:
No we do not have things like this in eventing.
I talked with Grace and I'm not sure why knative-gcp will need this, it's a bit hacky IMO, e.g. how do you guarantee users will have this patch?